src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/LocationFilter.tsx

```diff
@@ -0,0 +1,57 @@
+import { Checkbox, Flex } from "@artsy/palette"
+import * as React from "react"
+import { ShowMore } from "Components/ArtworkFilter/ArtworkFilters/ShowMore"
+import {
+  useAuctionResultsFilterContext,
+  useCurrentlySelectedFiltersForAuctionResults,
+} from "Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext"
+import { FilterExpandable } from "Components/ArtworkFilter/ArtworkFilters/FilterExpandable"
+
+export const locationMap = [
+  { displayName: "New York", name: "New York" },
+  { displayName: "Shanghai", name: "Shanghai" },
+  { displayName: "London", name: "London" },
+  { displayName: "Paris", name: "Paris" },
+  { displayName: "Vienna", name: "Vienna" },
+  { displayName: "Amsterdam", name: "Amsterdam" },
+]
```

Comment:
Wouldn't we want to be pulling these from the actual results? I don't think we want a hardcoded list?